tests/CMakeLists.txt

```
@@ -74,17 +92,35 @@ set(OPENSHOT_TEST_FILES
   KeyFrame_Tests.cpp
   Point_Tests.cpp
   Settings_Tests.cpp
-  Timeline_Tests.cpp )
+  Timeline_Tests.cpp)
+
+##########  SET OPENCV RELATED TEST FILES  ###############
+set(OPENSHOT_CV_TEST_FILES
+  CVTracker_Tests.cpp
+  CVStabilizer_Tests.cpp
+  # CVObjectDetection_Tests.cpp
+  )
```

```suggestion
##########  SET OPENCV RELATED TEST FILES  ###############
if(OpenCV_FOUND)
  set(OPENSHOT_CV_TEST_FILES
    CVTracker_Tests.cpp
    CVStabilizer_Tests.cpp
    # CVObjectDetection_Tests.cpp
    )
  set(OPENSHOT_CV_LIBRARIES
    ${OpenCV_LIBS}
    ${PROTOBUF_LIBRARY}
  )
endif()
```
Then you can just use those variables unconditionally in the executable definition. CMake will expand any undefined variable to an empty string.